query-engine/query-structure/src/default_value.rs

```diff
@@ -182,12 +182,20 @@ impl ValueGenerator {
         ValueGenerator::new("now".to_owned(), vec![]).unwrap()
     }
 
-    pub fn new_cuid() -> Self {
-        ValueGenerator::new("cuid".to_owned(), vec![]).unwrap()
+    pub fn new_cuid(version: u8) -> Self {
+        ValueGenerator::new(
+            "cuid".to_owned(),
+            vec![(Some(format!("{version}")), PrismaValue::Int(version as i64))],
```

Comment:
nit:
```suggestion
            vec![(Some(version.to_string()), PrismaValue::Int(version as i64))],
```

Reply:
Even better, this should be `None`, according to how `nanoid` was implemented: https://github.com/prisma/prisma-engines/pull/5047/commits/5102928a31dd9deabecd6353b8922c7f0d67e649#